.github/workflows/merge.yml

```diff
@@ -13,11 +13,11 @@ jobs:
     name: Conventional Commits
     runs-on: ubuntu-latest
     steps:
-      - if: github.ref == 'refs/heads/staging'
-        uses: actions/checkout@v3
       - if: github.ref == 'refs/heads/staging'
         uses: actions/setup-node@v3
       - if: github.ref == 'refs/heads/staging'
-        run: |
-          npm install @commitlint/config-conventional
-          echo "${{ github.event.head_commit.message }}" | npx commitlint --extends @commitlint/config-conventional
+        run: npm install @commitlint/config-conventional
+      - if: github.ref == 'refs/heads/staging'
```

Comment:
Is `staging` a bors thing?

Reply:
Yes, `staging` is a branch used by bors to check merge commits.|